deploy/images/Makefile

```diff
@@ -1,7 +1,8 @@
 SHELL := /bin/bash
 
 TAG=$(shell git rev-parse --short HEAD)
-
+BUILD_DATE=$(shell date -u +'%Y-%m-%dT%H:%M:%SZ')
+SOURCE_COMMIT=$(shell git rev-parse --short HEAD)
```

Comment:
SOURCE_COMMIT=TAG, any reason to use both?

Reply:
Not really needed but travis had a hiccup when the shell commands where inline in the docker build command.